src/core/cli/lifecycle/startlock.go

```
@@ -0,0 +1,41 @@
+	default:
+		return fn()
+	}
+
+	if err := os.MkdirAll(filepath.Dir(lockPath), 0755); err != nil {
+		return err
+	}
+	f, err := os.OpenFile(lockPath, os.O_CREATE|os.O_RDWR, 0644)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `f.Close` is not checked

(errcheck)